Signal/src/views/OWSSystemMessageCell.h

```diff
@@ -21,6 +23,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 @property (nonatomic, weak) id<OWSSystemMessageCellDelegate> systemMessageCellDelegate;
 
+@property (nonatomic, readonly) UILabel *titleLabel;
```

Comment:
using this to set the target rect for the menu, otherwise the contentCell might be a bit too high when we're showing a date stamp.